src/container/srv_target.c

```diff
@@ -1820,7 +1823,8 @@ ds_cont_tgt_epoch_aggregate_handler(crt_rpc_t *rpc)
 	if (out->tao_rc != 0)
 		return;
 
-	rc = dss_thread_collective(cont_epoch_aggregate_one, NULL, 0);
+	rc = dss_thread_collective(cont_epoch_aggregate_one, NULL, 0,
```

Comment:
seems the above cont_snap_notify_one and cont_epoch_aggregate_one can be in DSS_ULT_AGGREGATE pool?

Reply:
oh, DSS_ULT_AGGREGATE will be put to main xstream in sched_ult2xs, but here we want to create the xstream on each vos xstream. 

But indeed, these definition and sched_ult2xs and schedule_ult2pool is a bit inconsistency, probably need cleanup in different patch.